### PR TITLE
Add rails pack with Ruby 2.7.5

### PR DESCRIPTION
**Pull request by zuccamia**
Simply a build pack using `ruby:2.7.5` with NodeJS installed for `bundle` and `assets` images of some of our recently updated repos (e.g HCS)

## How to Test
Watching CI pass. (check for this image:  https://github.com/degica/dockerfiles/runs/7224591667?check_suite_focus=true)
[HCS](https://github.com/degica/hcs/pull/497/commits/6a5a5d54c265bd6137050e505ebeb6ab6b69a585) will use this image to bundle gems and precompile assets. This image is essentially the same with what HCS built with successfully previously (both pulling from the official `ruby:2.7.5` image) but with added NodeJS and a bunch of other general dependencies same as our other ruby images (not necessary for HCS but to have a ready image for others to pull too).

### CODE REVIEW

File: rails-buildpack/2.7.5/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+      liblzma-dev \
+      libmagickcore-dev \
+      libmagickwand-dev \
+      libreadline-dev \
+      libtool \
+      libxslt-dev \
+      libpq-dev \
+      libsqlite3-dev \
+      libxml2-dev \
+      qtbase5-dev qtchooser qt5-qmake qtbase5-dev-tools \
```
Comment (zuccamia) [inline rails-buildpack/2.7.5/Dockerfile:29]: `ruby:2.7.5` is a full image that's usually [based on](https://medium.com/swlh/alpine-slim-stretch-buster-jessie-bullseye-bookworm-what-are-the-differences-in-docker-62171ed4531d) the latest stable Debian OS. It seems from at least Debian 10.4 release, `qt4default` [has been deprecated](https://packages.debian.org/buster/qt5-default) in favor of more explicit dependencies build.

**Comment by degicat** [comment]
@yunshine can you help us review this PR, please?

File: rails-buildpack/2.7.5/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+ && rm -rf /var/lib/apt/lists/*
+
+# Install Node JS 16.x
+RUN curl -sL https://deb.nodesource.com/setup_lts.x | bash - \
+   && apt-get install -y nodejs \
+   && rm -rf /var/lib/apt/lists/*
+RUN npm install -g yarn
+
+ENV LANG C.UTF-8
+ENV LC_ALL C.UTF-8i
```
Comment (yunshine) [inline rails-buildpack/2.7.5/Dockerfile]: @zuccamia Out of curiosity, is there a reason you chose to go with UTF-8i? The Dockerfiles in other versions of rails-buildpack seem to be going with UTF-8, so I was just wondering!

Comment (zuccamia) [reply rails-buildpack/2.7.5/Dockerfile]: Whoa that's really good catch! Thanks @yunshine! It was a typo 😔 I fixed it in dd5bd74
Could you take another look when you have time? 🙇

**Review by yunshine — APPROVED** [review]
Thanks for all of your work on this! I learned a lot from all the links and articles you added to this PR. CI passes, and LGTM! 👍🏻

**Pull request merged**